thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/translator/DetectionConfigTranslator.java

```diff
@@ -530,6 +535,11 @@ private DetectionConfigDTO generateDetectionConfig(Map<String, Object> yamlConfi
     config.setActive(MapUtils.getBooleanValue(yamlConfigMap, PROP_ACTIVE, true));
     config.setYaml(yamlConfig);
 
+    //TODO: data-availability trigger is only enabled for detections running on PINOT only
```

Comment:
Why this is a TODO?

Reply:
Ideally, data availability trigger should be enabled if the cron is specified. However, due to the current prod setup, we only get events for Pinot dataset. We should remove the check for Pinot dataset when we get update events for other data source.

Reply:
Let's not add TODO here since this is a new feature request not a code refactor or tech debt. :)
We don't have ETA for getting event data for other datasets.